app/test/e2e/vocabulary-dataset.spec.js

```diff
@@ -167,14 +149,65 @@ describe('Vocabulary-dataset relationships test suite', () => {
             .post(`/api/v1/dataset/${mockDatasetId}/vocabulary`)
             .send({ loggedUser: USERS.ADMIN });
 
-        // Assert the response as 200 OK
-        response.status.should.equal(200);
-        response.body.should.have.property('data').and.be.an('array');
+        assert200(response);
         response.body.data[0].attributes.should.have.property('name').and.equal('sciencev3');
         response.body.data[0].attributes.should.have.property('application').and.equal('rw');
         response.body.data[0].attributes.should.have.property('tags').and.deep.equal(['biology', 'chemistry']);
     });
 
+    it('Using PUT, PATCH and DELETE to manage vocab-dataset relationships with auth returns 200 OK with data', async () => {
```

Comment:
split this up please

Reply:
Broke this test into 3 single-purposed tests!